### ISSUE TITLE
question: will this book be available in print once it's completed?

### ISSUE THREAD

**Issue by jrheard** [skerkour/black-hat-rust#8]
hi! i like to read physical copies of programming books so that i can mark them up and keep them on my shelf. will https://academy.kerkour.com/black-hat-rust?coupon=BLOG be available in print form once it's completed? i didn't see an obvious answer on the marketing website or on this github repo, please let me know if i missed something. thanks very much, best of luck with the book, looks really cool!

**Comment by mwaitzman** [comment]
Unless the answer has changed since he answered this for me about 3 months ago, no.

**Comment by skerkour** [comment]
Hello,
Thank you for reaching out!

I'm open to a physical edition, but here are a few observations:

From what I understand, Self-publishing a paper edition is now easy thanks to Amazon KDP.

The problem with physical editions is that everything needs to be perfect as they can't be edited for mistakes.

Paper editions seem not to be a great fit for code, but, on the other hand, it's easier to have a paper book on the desk and read / write code on a computer screen at the same time.


**TL;DR:** This is not the focus of my attention today, but I will leave this issue open so the persons interested in a paper edition can upvote it.

Once the digital edition is finalized, I will reach you back to see what we can do :)

Feel free to add your observations about the advantages of a paper edition over a digital one, so I can better understand the need.

**Comment by skerkour** [comment]
PS: For the people interested in a paper edition, please feel free to subscribe to this issue (on the right)

**Comment by jrheard** [comment]
great, thanks very much! in addition to just enjoying the physical sense of holding and marking up a book and having it on my shelf to look at, another thing that i particularly like about paper books is that i don't have to worry about losing my place, because physical bookmarks are pretty foolproof; i also like that a paper book focuses my attention, because i'm not looking at the book on a device and so i can't just open up another browser tab and get distracted from the book. self-control is difficult :)

i totally get what you mean re: everything having to be perfect, i'd definitely find that aspect frustrating/intimidating if i were in your shoes. i also can't imagine what laying out code examples must be like, it does sound like a nightmare.

thanks again! :)

**Comment by captain-yossarian** [comment]
@skerkour as for the mistakes in paper edition. Dont worry sbout them. I'd willing to bet that your book is good, this is the most important. Is it possible to print small amout of books for us? I 'm ready to pay more.

**Comment by diversable** [comment]
Dear Captain-y...

Perhaps yours and your friends' small pot of money could fund a small group of lowly copy editors (whether professonals or volunteers moonlighting) to give the 'final' digital version a once-over before it gets published, which might give the author an incentive to get on board, as it were..

**Comment by captain-yossarian** [comment]
@diversable Hi, this is nice idea. However, I'm from Ukraine and I went back to Ukraine to fight for my country against russian b*stards. All my money will go to Ukraine army account.
